my_list.rb

```diff
@@ -0,0 +1,11 @@
+class MyList
+  include Enumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Kindly consider creating the `each` method on your own instead of using the default `each` method